go/exorelay/test-fixtures/register_handler/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
```

Comment:
I would add a little comment to these test fixtures explaining what their function is.

Reply:
Added above the main functions